examples/src/Viewer.tsx

```diff
@@ -477,18 +495,17 @@ class Viewer extends React.Component<InputParams, ViewerState> {
             this.receiveConvertedFile();
         }
         // NOTE: Currently incorrectly assumes initial time of 0
-        const totalDuration = (data.totalSteps - 1) * data.timeStepSize;
         this.setState({
-            totalDuration,
+            totalSteps: data.totalSteps,
             timeStep: data.timeStepSize,
             currentFrame: 0,
             currentTime: 0,
             trajectoryTitle: data.trajectoryTitle,
         });
     }
 
-    public handleScrubTime(event): void {
-        simulariumController.gotoTime(parseFloat(event.target.value));
+    public handleScrubFrame(event): void {
+        simulariumController.movePlaybackFrame(parseInt(event.target.value));
```

Comment:
`goToFrame` seems a lot simpler, and would mirror the `goToTime` function. I haven't fully ingested why you decided to move to frames, but I could see an argument for keeping both